src/forms/subscriptions.ts

```diff
@@ -362,12 +363,12 @@ class FormFields extends BaseFormFields {
         return this.builder.getFieldLabelByName(SubscriptionFields.SubSelectName);
     }
 
-    getSubTriggerByID(subID: string): any {
-        const triggers = this.getChannelTriggers(this.getSelectedChannelID());
-        return triggers.find((trigger) => String(trigger.id) === subID);
+    getSubTriggerByID(subID: string): ZDTrigger {
+        const triggers: ZDTrigger[] = this.getChannelTriggers(this.getSelectedChannelID());
+        return triggers.find((trigger) => String(trigger.id) === subID) as ZDTrigger;
```

Comment:
Are we guaranteed to find a trigger with this id here?

Reply:
converted to throw an error